arviz/plots/ppplot.py

```diff
@@ -0,0 +1,169 @@
+    textsize: float
+        Text size scaling factor for labels, titles and lines. If None it will be
+        autoscaled based on figsize.
+    var_names : list
+        List of variables to be plotted.
+    coords : dict
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for
+        that dimension.
+    transform : callable
+        Function to transform data (defaults to None i.e. the identity function)
+    data_pairs : dict
+        Dictionary containing relations between prior data and posterior data.
+        Dictionary structure:
+
+        - key = prior data var_name
+        - value = posterior var_name
+
+        For example, `data_pairs = {'y' : 'y_hat'}`
+        If None, it will assume that the prior data and the posterior
+        data have the same variable name.
+    legend : bool
+        Add legend to figure. By default True.
+    ax: axes, optional
+        Matplotlib axes: a numpy 2d array of matplotlib axes.
+    prior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for prior group.
+    posterior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for posterior group.
+    backend: str, optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    backend_kwargs: bool, optional
+        These are kwargs specific to the backend being used. For additional documentation
+        check the plotting method of the backend.
+    show : bool, optional
+        Call backend show function.
+
+    Returns
+    -------
+    axes : matplotlib axes or bokeh figures
+
+    Examples
+    --------
+    Plot the prior/posterior plot for specified vars and coords.
+
+    .. plot::
+        :context: close-figs
+
+        >>> import arviz as az
+        >>> data = az.load_arviz_data('radon')
+        >>> az.plot_pp(data, var_names=["defs"], coords={"team" : ["Italy"]})
+
+    """
+
+    groups = ["prior", "posterior"]
+
+    if transform is not None:
+        data = transform(data)
+
+    if coords is None:
+        coords = {}
+
+    if prior_kwargs is None:
+        prior_kwargs = {}
+
+    if posterior_kwargs is None:
+        posterior_kwargs = {}
+
+    if backend_kwargs is None:
+        backend_kwargs = {}
+
+    if data_pairs is None:
+        data_pairs = {}
+
+    datasets = [getattr(data, group) for group in groups]
+
+    if var_names is None:
+        var_names = list(datasets[0].data_vars)
+    var_names = _var_names(var_names, datasets[0])
+    vars = [var_names]
+    for i in range(1, len(datasets)):
+        var_name =[data_pairs.get(var, var) for var in var_names]
+        var_name = _var_names(var_name,  datasets[i])
+        vars.append(var_name)
+
+
+    pp_plotters = [
+        list(xarray_var_iter(get_coords(data, coords), var_names=var_names, combined=True))
+        for data , var in zip(datasets, vars)
+    ]
```

Comment:
Make sure `coords` is a list of same length as datasets (like in forestplot) and zip also on coords `... for data, var, coords_i in zip(datasets, vars, coords)`

Reply:
I can't seem to understand, why is this necessary after the above changes?

Reply:
that would be to use different coordinates in the different datasets

Reply:
Currently, `coords` are selected using:
```
datasets = get_coords(
        datasets, list(reversed(coords)) if isinstance(coords, (list, tuple)) else coords
    )
```
After this, I don't think we need to apply the above changes. I also tested `az.plot_pp(data, var_names=["defs"], coords=[{"team" : [ "France"]}, {"team" : [ "Italy"]}])` and its working fine. Am I mistaking something?

Reply:
I am not sure we expect the same behaviour :thinking:. Imagine you have an idata with posterior group containing only a variable `defs` with coords `defs_dim_0: [0, 1, 2]` and prior group containing only a variable `defs_prior` with coords `defs_prior_dim_0: [0, 1, 2]` (it is common to have different names between groups even though they may be the same var, and these are the default dim names, could be `team` and `team_prior`, the point is that coord name could be different too. Would is be possible to compare posterior and prior only for coord==0?

Reply:
True, it should work now!